firmware/boron-ins-fsm/src/im21door.cpp

```diff
@@ -69,7 +70,6 @@ doorData checkIM21(){
     // Checks if the 2nd bit (counting from 0) of doorStatus is 1
     // read as: doorLowBatteryFlag is true if doorStatus AND 0b0100 is not 0b0000
     doorLowBatteryFlag = (currentDoorData.doorStatus & (1 << 2)) != 0;
-
```

Comment:
I like the idea of having all functions fit on one screen. So I 'automatically' remove newlines if I feel like a pre-existing comment(s) separate the two sections just fine. I can add it back if you'd like.

Reply:
So funny. Different people's version of readability. I find the extra lines help me not become bogged down in the text, but I can also understand the advantage of fitting everything on one screen.

In general, I'd say try to stick with the conventions of the codebase that you're in. So if there are newlines, then they should stay. And if there aren't newlines, then they shouldn't be added. It also makes reviewing the diff easier because then less stuff changes. Thanks!

Reply:
Gotcha, I'll remove this change.